tools/shared.py

```diff
@@ -1627,7 +1627,7 @@ def llvm_as(input_filename, output_filename=None):
   nm_cache = {} # cache results of nm - it can be slow to run
 
   @staticmethod
-  def llvm_nm(filename, stdout=PIPE, stderr=None):
+  def llvm_nm(filename, stdout=PIPE, stderr=None, **kwargs):
```

Comment:
is there a benefit to using `kwargs`? It could just be `include_internal=False`.